ajax/libs/hls.js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hls.js",
-  "version": "0.12.4",
+  "version": "0.13.0",
```

Comment:
Please include this version and any missing versions in this PR.

Reply:
0.12.4 is already on cdnjs and 0.13.0 is (apart from some beta's) the only missing version.
So no changes required or do I oversee something?
Thanks :)

Reply:
I have also included the missing 0.13.0 files now.
Please let me know if this is OK :)

Reply:
Is it ready to be merged now?
Would be nice 😊